boilerplate/openshift/golang_osd_hive_operator/.codecov.yml

```diff
@@ -0,0 +1,26 @@
+codecov:
```

Comment:
Can we DRY this up? I hate to see copied files like this. I was hoping for putting all of these common files into an `openshift/golang_osd_operator_common` convention or something.

Reply:
We've talked around this extensively over the past couple of days and come up with some great ideas for commonizing, dependency management, etc.

Those things are eminently doable, but a nontrivial amount of extra work that we would like to defer for the time being.

We also talked about it early on in the design process and decided the same thing.

The maintenance is pretty much `cp -r`. The test lets you know when you missed doing that.